server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -345,4 +347,23 @@ public void jmapUserTasksShouldBeExposed() throws Exception {
             .body("status", is("completed"))
             .body("type", is("RecomputeUserFastViewProjectionItemsTask"));
     }
+
+    @Test
+    public void jmapMessageTasksShouldBeExposed() {
+        String messageId = getMessageId();
+
+        String taskId = with()
+            .queryParam("task", "recomputeFastViewProjectionItems")
+            .post("/messages/" + messageId)
+            .jsonPath()
+            .get("taskId");
+
+        given()
+            .basePath(TasksRoutes.BASE)
+        .when()
+            .get(taskId + "/await")
+        .then()
+            .body("status", is("failed"))
```

Comment:
Why failed? Can we add probes to create a "real" message that we can successfully reIndex?

Reply:
Task is failed because the message doesn't exist, that's the behavior. The test is about the task being exposed, as its name, but I agree with you it sounds better to add a message before, to successfully recompute its preview. I will check

Reply:
IMO it makes more sense to demonstrate a working behaviour in an integration test than a failing one.

Here  you are just demonstrating that we are failing correctly, this is not what integration tests are meant to.